File: JavaScript/8js 1/capitalizeWord.js
/*
|--------------------------------------------------------------------------
| Capitalization and Mutability
|--------------------------------------------------------------------------
|
*/

function capitalizeWord(word) {
   return word[0].toUpperCase() + word.slice(1, word.length).toLowerCase();
}


function capitalizeWord(word) {
   return word.replace(/^\w/, (val) => val.toUpperCase());
}


function capitalizeWord(word) {
   return word[0].toUpperCase() + word.slice(1);
}


function capitalizeWord(word) {
   return `${word[0].toUpperCase()}${word.slice(1)}`;
}


console.log(capitalizeWord("word"), "Word");
console.log(capitalizeWord("i"), "I");
console.log(capitalizeWord("glasswear"), "Glasswear");

File: Python/8py 1/capitalize_word.py
"""
|--------------------------------------------------------------------------
| Capitalization and Mutability
|--------------------------------------------------------------------------
|
"""

def capitalize_word(word):
   return word.capitalize()


def capitalizeWord(word):
   return word[:1].upper() + word[1:]


def capitalizeWord(s):
   return s.title()


print(capitalize_word('word'), 'Word')
print(capitalize_word('i'), 'I')
print(capitalize_word('glasswear'), 'Glasswear')